src/appmixer/kafkadavid/platform/SendMessage/SendMessage.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+module.exports = {
+
+    start(context) {
+
+        const { flowId, componentId, auth } = context;
+        return context.callAppmixer({
+            endPoint: '/plugins/appmixer/kafkadavid/producers',
+            method: 'POST',
+            body: {
+                flowId,
+                componentId,
+                auth
+            }
+        });
+    },
+
+    async stop(context) {
+
+        return context.callAppmixer({
+            endPoint: `/plugins/appmixer/kafkadavid/producers/${context.flowId}/${context.componentId}`,
+            method: 'DELETE'
+        });
+    },
+
+    async receive(context) {
+
+        const {
+            topic,
+            key,
+            value,
+            acks,
+            timeout,
+            compression,
+            partition,
+            timestamp,
+            headers
+        } = context.messages.in.content;
+
+        const payload = {
+            topic,
+            messages: [
+                {
+                    key,
+                    value,
+                    partition,
+                    timestamp: timestamp && new Date(timestamp).getTime(),
+                    headers: headers && JSON.parse(headers)
+                }
+            ],
+            acks,
+            timeout
+        };
+
+        await context.callAppmixer({
```

Comment:
Scenario:
- a new pod is added, or engine is restarted
- message from Rabbit is consumed and gets here
- the job in Kafka plugin did not run yet, it's scheduled to run every minute - therefore the connection is not created
- POST to /send 
- the connections.sendMessage is triggered and there is a condition ` if (!producer) return; // Connection doesn't exist, do nothing` and your message is lost forever.